.drone.star

```diff
@@ -366,6 +492,260 @@ def phan():
 
 	return pipelines
 
+def litmus():
```

Comment:
probably this is not called and can be deleted

Reply:
It has been called in `stagePipelines`